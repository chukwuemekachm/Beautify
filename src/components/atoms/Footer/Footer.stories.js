// Modules
import React from 'react';
import { storiesOf } from '@storybook/react';
import { withInfo } from '@storybook/addon-info';
import { mount } from 'enzyme';

// Footer
import Footer from './Footer';

const stories = storiesOf('Footer', module);
stories.addDecorator(withInfo);

stories.add('Default Footer', () => <Footer />);
