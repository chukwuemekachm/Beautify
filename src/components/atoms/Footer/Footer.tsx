// Modules
import * as React from 'react';
import styled from 'styled-components';

// Styles
import { dimGrey, white } from '../../../styles/colors';
import { nunitoSans } from '../../../styles/fonts';
import { mediumFont } from '../../../styles/sizes';
import remCalc from '../../../utils/remCalc';

/**
 * @description The Footer Molecule
 */
const Footer: React.SFC<{}> = () => (
  <StyledFooter>MADE WITH LOVE BY TEAM BEAUTIFY</StyledFooter>
);

const StyledFooter = styled.h3`
  font-family: ${nunitoSans};
  font-style: normal;
  font-weight: 900;
  line-height: normal;
  font-size: ${mediumFont};
  letter-spacing: ${remCalc(1.6)};
  color: ${white};
  text-align: center;
  background-color: ${dimGrey};
  padding-top: ${remCalc(48)};
  padding-bottom: ${remCalc(48)};
`;

export default Footer;
