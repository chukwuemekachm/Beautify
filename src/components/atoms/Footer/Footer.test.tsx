import { shallow } from 'enzyme';
import * as React from 'react';

import Footer from './Footer';

describe('Test Footer.tsx', () => {
  const wrapper = shallow(<Footer />);

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should contain the appropriate text', () => {
    expect(wrapper.text())
      .toBe('MADE WITH LOVE BY TEAM BEAUTIFY');
  });
});
