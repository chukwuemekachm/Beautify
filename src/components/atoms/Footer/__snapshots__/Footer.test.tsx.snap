// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer.tsx should match snapshot 1`] = `
<styled.h3>
  MADE WITH LOVE BY TEAM BEAUTIFY
</styled.h3>
`;
